packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -767,6 +770,11 @@ document.getElementById('resume-recording').addEventListener('click', () => {
   }
 });
 
+document.getElementById('upload-logs').addEventListener('click', () => {
```

Comment:
Should you pass `event` as parameter?
```suggestion
document.getElementById('upload-logs').addEventListener('click', (event) => {
```